tests/test_name_server.rb

```diff
@@ -44,8 +44,8 @@ def no_nameserver_google
   # TESTS
 
   def test_nameserver_create_destroy_single_ipv4
-    id = '8.8.8.8'
-    refute_includes(Cisco::NameServer.nameservers, id)
+    id = '7.7.7.7'
+    assert_equal({}, Cisco::NameServer.nameservers)
```

Comment:
I am in favor of removing any pre-existent config before running the tests.  This is the only way to ensure predictability when running the tests against physical and virtual devices.